flink-libraries/flink-table/src/main/scala/org/apache/flink/api/table/AbstractTableEnvironment.scala

```diff
@@ -83,4 +84,17 @@ class AbstractTableEnvironment {
     )
     TranslationContext.registerTable(dataSetTable, name)
   }
+
+  /**
+   * Execute a SQL query on a batch [[Table]].
+   * The [[Table]] has to be registered in the tables registry
```

Comment:
We can query more than one table (all tables need to be registered). Tables should be registered at the TableEnvironment. I think the term `tables registry` is not defined.